dandischema/tests/test_conf.py

```diff
@@ -40,7 +43,7 @@ def test_valid_instance_name(self, instance_name: str) -> None:
         """
         from dandischema.conf import Config
 
-        Config(instance_name=instance_name)
+        Config(dandi_instance_name=instance_name)
```

Comment:
so we are loosing convenience of automated `dandi_` `env_prefix` entirely and need to duplicate it every time we would instantiate `Config`?